skills/synology-nas/references/shares-nfs.md

```
@@ -83,13 +86,13 @@ sudo mount -t nfs <nas-ip>:/volume1/Photos /mnt/photos -o ro,nfsvers=4
 
 ### "Open NFS access to the whole subnet"
 
-Use a CIDR client string:
+Use a CIDR string for `client_ip`:
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)